packages/help-center/src/components/help-center-contact-form.tsx

```diff
@@ -289,6 +290,12 @@ export const HelpCenterContactForm = () => {
 						`Plan: ${ productId } - ${ productName } (${ productTerm })`,
 					];
 
+					if ( getQueryArgs()?.ref === 'woocommerce-com' ) {
+						ticketMeta.push(
+							`Created: during store setup on ${ isWcMobileApp() ? 'Woo mobile app' : 'browser' }`
```

Comment:
Nitpicky, but I'd prefer this to follow the original P2 comment since it will later be used for string search in ZD:
```suggestion
							`Created during store setup on ${ isWcMobileApp() ? 'Woo mobile app' : 'Woo browser' }`
```

Reply:
Granted, `Woo browser` is not a thing, but I digress 😐

Reply:
Thanks! 😆